testing/tests/DevExpress.ui.widgets.form/form.markup.tests.js

```diff
@@ -773,6 +776,202 @@ QUnit.module('Form', () => {
     });
 });
 
+QUnit.module('Class check', ()=>{
```

Comment:
We can specify a class name here and don't remind it in each test 
```suggestion
QUnit.module(`"${TOGGLE_CONTROLS_PADDING_CLASS}" class`, ()=>{
```

Reply:
Fixed